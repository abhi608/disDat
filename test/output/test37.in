Starting T1
Starting T2
Starting read only transaction T3
=== Site 1 ===
All other variables have same initial value
=== Site 2 ===
All other variables have same initial value
=== Site 3 ===
All other variables have same initial value
=== Site 4 ===
All other variables have same initial value
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
Starting read only transaction T4
T2 committed
Starting read only transaction T5
T3 committed
T4 committed
T5 committed
=== Site 1 ===
All other variables have same initial value
=== Site 2 ===
All other variables have same initial value
=== Site 3 ===
All other variables have same initial value
=== Site 4 ===
All other variables have same initial value
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
Starting T6
Starting T7
Starting T8
T7 got write lock on x6
T6 got write lock on x8
T8 got write lock on x10
T7 committed
DEBUG| Clearing site locks for T7 variable: x6
Clearing locks for T7 variable: x6
T6 committed
DEBUG| Clearing site locks for T6 variable: x8
Clearing locks for T6 variable: x8
=== Site 1 ===
x8: 200 at site 1
x6: 20 at site 1
All other variables have same initial value
=== Site 2 ===
x8: 200 at site 2
x6: 20 at site 2
All other variables have same initial value
=== Site 3 ===
x8: 200 at site 3
x6: 20 at site 3
All other variables have same initial value
=== Site 4 ===
x8: 200 at site 4
x6: 20 at site 4
All other variables have same initial value
=== Site 5 ===
x8: 200 at site 5
x6: 20 at site 5
All other variables have same initial value
=== Site 6 ===
x8: 200 at site 6
x6: 20 at site 6
All other variables have same initial value
=== Site 7 ===
x8: 200 at site 7
x6: 20 at site 7
All other variables have same initial value
=== Site 8 ===
x6: 20 at site 8
x8: 200 at site 8
All other variables have same initial value
=== Site 9 ===
x8: 200 at site 9
x6: 20 at site 9
All other variables have same initial value
=== Site 10 ===
x8: 200 at site 10
x6: 20 at site 10
All other variables have same initial value
Starting T9
Starting T10
Starting T11
Site: 1 failed
T8 aborted as site 1 failed
Site: 2 failed
T8 aborted as site 2 failed
Site: 3 failed
T8 aborted as site 3 failed
DEBUG| Clearing site locks for T8 variable: x10
Clearing locks for T8 variable: x10
T9 committed
Starting read only transaction T12
Site: 4 failed
Site: 5 failed
Site: 6 failed
Site: 7 failed
Site: 8 failed
Site: 9 failed
Site: 10 failed
Site: 5 recovered
