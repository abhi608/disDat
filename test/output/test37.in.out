Starting T1
Starting T2
Starting read only transaction T3
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 20 at site 1
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 20 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 20 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 60 at site 4
x4: 40 at site 4
x13: 130 at site 4
x8: 80 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 20 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 20 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 20 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 20 at site 10
T1 got write lock on x2
Instruction can't be recognized
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 write the value 100 of variable x2
T1 committed
DEBUG| Clearing site locks for T1 variable: x2
Clearing locks for T1 variable: x2
Starting read only transaction T4
T2 got write lock on x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 write the value 50 of variable x2
T2 committed
DEBUG| Clearing site locks for T2 variable: x2
Clearing locks for T2 variable: x2
Starting read only transaction T5
T3 committed
T4 committed
T5 committed
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 50 at site 1
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 50 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 50 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 60 at site 4
x4: 40 at site 4
x13: 130 at site 4
x8: 80 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 50 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 50 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 50 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 50 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 50 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 50 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 50 at site 10
Starting T6
Starting T7
Starting T8
T7 got write lock on x6
T6 got write lock on x8
T8 got write lock on x10
DEBUG| T6 did not get read lock on x6 site: 1
DEBUG| T6 did not get read lock on x6 site: 2
DEBUG| T6 did not get read lock on x6 site: 3
DEBUG| T6 did not get read lock on x6 site: 4
DEBUG| T6 did not get read lock on x6 site: 5
DEBUG| T6 did not get read lock on x6 site: 6
DEBUG| T6 did not get read lock on x6 site: 7
DEBUG| T6 did not get read lock on x6 site: 8
DEBUG| T6 did not get read lock on x6 site: 9
DEBUG| T6 did not get read lock on x6 site: 10
T6 is blocked by T7 on x6
DEBUG| T7 did not get read lock on x10 site: 1
DEBUG| T7 did not get read lock on x10 site: 2
DEBUG| T7 did not get read lock on x10 site: 3
DEBUG| T7 did not get read lock on x10 site: 4
DEBUG| T7 did not get read lock on x10 site: 5
DEBUG| T7 did not get read lock on x10 site: 6
DEBUG| T7 did not get read lock on x10 site: 7
DEBUG| T7 did not get read lock on x10 site: 8
DEBUG| T7 did not get read lock on x10 site: 9
DEBUG| T7 did not get read lock on x10 site: 10
T7 is blocked by T8 on x10
DEBUG| T6 did not get read lock on x6 site: 1
DEBUG| T6 did not get read lock on x6 site: 2
DEBUG| T6 did not get read lock on x6 site: 3
DEBUG| T6 did not get read lock on x6 site: 4
DEBUG| T6 did not get read lock on x6 site: 5
DEBUG| T6 did not get read lock on x6 site: 6
DEBUG| T6 did not get read lock on x6 site: 7
DEBUG| T6 did not get read lock on x6 site: 8
DEBUG| T6 did not get read lock on x6 site: 9
DEBUG| T6 did not get read lock on x6 site: 10
T6 is blocked by T7 on x6
DEBUG| T8 did not get read lock on x8 site: 1
DEBUG| T8 did not get read lock on x8 site: 2
DEBUG| T8 did not get read lock on x8 site: 3
DEBUG| T8 did not get read lock on x8 site: 4
DEBUG| T8 did not get read lock on x8 site: 5
DEBUG| T8 did not get read lock on x8 site: 6
DEBUG| T8 did not get read lock on x8 site: 7
DEBUG| T8 did not get read lock on x8 site: 8
DEBUG| T8 did not get read lock on x8 site: 9
DEBUG| T8 did not get read lock on x8 site: 10
T8 is blocked by T6 on x8
T8 aborted as it is youngest in a deadlock
DEBUG| Clearing site locks for T8 variable: x10
Clearing locks for T8 variable: x10
DEBUG| T6 did not get read lock on x6 site: 1
DEBUG| T6 did not get read lock on x6 site: 2
DEBUG| T6 did not get read lock on x6 site: 3
DEBUG| T6 did not get read lock on x6 site: 4
DEBUG| T6 did not get read lock on x6 site: 5
DEBUG| T6 did not get read lock on x6 site: 6
DEBUG| T6 did not get read lock on x6 site: 7
DEBUG| T6 did not get read lock on x6 site: 8
DEBUG| T6 did not get read lock on x6 site: 9
DEBUG| T6 did not get read lock on x6 site: 10
T6 is blocked by T7 on x6
T7 got read lock on x10 having value 100
T7 read the value 100 of variable x10
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 write the value 20 of variable x6
T7 committed
DEBUG| Clearing site locks for T7 variable: x10
Clearing locks for T7 variable: x10
DEBUG| Clearing site locks for T7 variable: x6
Clearing locks for T7 variable: x6
T6 got read lock on x6 having value 20
T6 already has a read lock on x6
T6 got read lock on x10 having value 100
T6 got read lock on x8 having value 200
T6 read the value 200 of variable x8
T6 read the value 100 of variable x10
T6 read the value 20 of variable x6
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 write the value 200 of variable x8
T6 committed
DEBUG| Clearing site locks for T6 variable: x10
Clearing locks for T6 variable: x10
DEBUG| Clearing site locks for T6 variable: x6
Clearing locks for T6 variable: x6
DEBUG| Clearing site locks for T6 variable: x8
Clearing locks for T6 variable: x8
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 200 at site 1
x6: 20 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 50 at site 1
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 200 at site 2
x6: 20 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 50 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 200 at site 3
x6: 20 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 50 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 20 at site 4
x4: 40 at site 4
x13: 130 at site 4
x8: 200 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 50 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 200 at site 5
x6: 20 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 50 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 200 at site 6
x6: 20 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 50 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 200 at site 7
x6: 20 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 50 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 200 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 20 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 50 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 200 at site 9
x6: 20 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 50 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 200 at site 10
x6: 20 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 50 at site 10
Starting T9
Starting T10
Starting T11
Site: 4 failed
T9 is waiting on x3
Site: 4 recovered
Site: 1 failed
Site: 2 failed
Site: 3 failed
Though the site holding x3 is recovering, T9 got read lock on x3 having value 30 since its the only copy
T9 got read lock on x2 having value 50
T9 already has a read lock on x3
T9 got write lock on x2
T9 read the value 50 of variable x2
T9 read the value 30 of variable x3
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 write the value 9 of variable x2
T9 committed
DEBUG| Clearing site locks for T9 variable: x2
Clearing locks for T9 variable: x2
DEBUG| Clearing site locks for T9 variable: x2
Clearing locks for T9 variable: x2
DEBUG| Clearing site locks for T9 variable: x3
Clearing locks for T9 variable: x3
T10 got read lock on x2 having value 9
Starting read only transaction T12
Site: 4 failed
T10 aborted as site 4 failed
Site: 5 failed
Site: 6 failed
Site: 7 failed
Site: 8 failed
Site: 9 failed
Site: 10 failed
DEBUG| Clearing site locks for T10 variable: x2
Clearing locks for T10 variable: x2
Site: 5 recovered
T11 got write lock on x2
T11 write the value 112 of variable x2
T11 committed
DEBUG| Clearing site locks for T11 variable: x2
Clearing locks for T11 variable: x2
